Description:  Fix validation issues
Authors: Nico Golde <nion@debian.org>, Siegfried-Angel Gevatter <rainct@ubuntu.com>

Index: obconf-2.0.3+git20110808/obconf.desktop
===================================================================
--- obconf-2.0.3+git20110808.orig/obconf.desktop	2011-08-05 18:05:10.000000000 +0200
+++ obconf-2.0.3+git20110808/obconf.desktop	2011-12-15 22:38:08.699538521 +0100
@@ -1,13 +1,19 @@
 [Desktop Entry]
-Encoding=UTF-8
+Version=1.0
 Type=Application
 Name=Openbox Configuration Manager
+Name[ca]=Gestor de configuració de l'Openbox
+Name[de]=Openbox Konfiguration Manager
+Name[es]=Gestor de configuración de Openbox
 Name[zh_TW]=Openbox 組態管理器
-Comment=Openbox configuration manager
+Comment=Configure and personalize the Openbox window manager
+Comment[ca]=Configureu i personalitzeu el gestor de finestres Openbox
+Comment[de]=Konfiguriert den Openbox Window Manager
+Comment[es]=Configura i personaliza el gestor de ventanas Openbox
 Comment[zh_TW]=管理 Openbox 組態設定
 Icon=obconf
 Exec=obconf %f
 Categories=Settings;DesktopSettings;GTK;
-MimeType=application/x-openbox-theme
+MimeType=application/x-openbox-theme;
 StartupNotify=true
 Terminal=false
Index: obconf-2.0.3+git20110808/x-openbox-theme.desktop
===================================================================
--- obconf-2.0.3+git20110808.orig/x-openbox-theme.desktop	2011-08-05 18:05:10.000000000 +0200
+++ obconf-2.0.3+git20110808/x-openbox-theme.desktop	2011-12-15 22:38:08.699538521 +0100
@@ -1,7 +1,8 @@
 [Desktop Entry]
-Encoding=UTF-8
+Version=1.0
 Type=MimeType
-MimeType=application/x-openbox-theme
+Name=Openbox Theme Archive
+MimeType=application/x-openbox-theme;
 Icon=openbox
 Patterns=*.obt;
 Comment=Openbox Theme Archive
